pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[project]
+name = "neo"
+version = "0.12.0.dev0"
+
+authors = [
```

Comment:
I think either we list _all_ authors, or we use the same approach as in the current setup.py, and give the name as "Neo authors and contributors"

Reply:
Sure. I forgot about that.